tests/cli/fluvio_smoke_tests/e2e-basic.bats

```diff
@@ -129,6 +136,11 @@ teardown_file() {
     assert_success
     run timeout 15s "$FLUVIO_BIN" topic create "$TOPIC_NAME_12"
     assert_success
+
+    if [[ "$FLUVIO_CLI_RELEASE_CHANNEL" == "stable" || "$FLUVIO_CLUSTER_RELEASE_CHANNEL" == "stable" ]]; then
```

Comment:
i think that this should be the opposite. Only run if not on `stable` (on `dev`)

Reply:
Applied on: [2187a36](https://github.com/infinyon/fluvio/pull/3185/commits/2187a36af39202a644ec75f388decf289d256ff0)